capstone/src/main/java/com/google/sps/data/User.java

```diff
@@ -32,6 +38,14 @@ public String getNickname() {
         return (String) this.delegate.getProperty(User.NICKNAME_PROPERTY_KEY);
     }
 
+    public void setEmbeddedTags(EmbeddedEntity entity) {
```

Comment:
As discussed, I would name this something other than "EmbeddedTags" which indicates what it is

Reply:
Got it.